webapp/src/main/java/org/apache/atlas/web/rest/EntityREST.java

```diff
@@ -1130,40 +1131,42 @@ public EntityAuditSearchResult searchAuditEvents(AuditSearchParams parameters) t
 
             EntityAuditSearchResult ret = esBasedAuditRepository.searchEvents(dslString);
 
-            scrubEntityAudits(ret);
+            scrubEntityAudits(ret, parameters.getSuppressLogs());
```

Comment:
parameters.getSuppressLogs() is always true
I think we do not need parameters.suppressLogs field 
Just pass true here

Reply:
This is meant for UI, If UI needs ranger audits , from UI   "suppressLogs" : false attribute can be sent.